packages/react-ui-kit/demo/Demo.jsx

```diff
@@ -135,12 +135,22 @@ class Demo extends React.PureComponent {
           </Modal>
         )}
         {this.state.isMenuModalOpen && (
-          <MenuModal onBackgroundClick={() => this.setState({isMenuModalOpen: false})}>
-            <MenuItem onClick={() => this.setState({isMenuModalOpen: false})}>Like</MenuItem>
-            <MenuItem onClick={() => this.setState({isMenuModalOpen: false})}>Edit</MenuItem>
-            <MenuItem onClick={() => this.setState({isMenuModalOpen: false})}>Delete for me...</MenuItem>
-            <MenuItem onClick={() => this.setState({isMenuModalOpen: false})}>Delete for everyone...</MenuItem>
-            <MenuItem onClick={() => this.setState({isMenuModalOpen: false})}>Cancel</MenuItem>
+          <MenuModal data-uie-name="should-be-there" onBackgroundClick={() => this.setState({isMenuModalOpen: false})}>
+            <MenuItem data-uie-name="should-be-there" onClick={() => this.setState({isMenuModalOpen: false})}>
```

Comment:
How about extracting this method into a constant and just give the constant to the click handler.

```
const closeModal = () => this.setState({isMenuModalOpen: false});

//...

<MenuItem onClick={closeModal}>..</MenuItem>
```
This would avoid creating 5 times the same function